INFO     root:Log.py:79 [INFO 2019-11-03 20:23:02]初始化关联文件
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:23:02]读取初始关联文件内容：   ItemsView(<Section: test_data>)
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:23:02]初始关联内容数据处理后：   {'name': '2', 'type': 'Web', 'version': 'h0PZMk37vs', 'description': 'api_auto_Q2IJ3jDB7t', 'project_id': '623116cf4d274453ba803e753fda2e69', 'token_id': '14be2548e67d4e1986d1ad542dfba7cf', 'project_name': 'ygKSbMNFRTdOcEtwPm8L', 'network_name': 'JYzSZPIFibRTMwsGtOXL_netwrok', 'subnet_name': 'dipXQNF9qwJe6k8WGV2O_subnet', 'network_id': 'af23f0b5-978f-4f65-9c8c-ed7685b92bc4', 'subnet_id': 'd4d29468-e3f8-4471-83d0-c908424ebb4f', 'image_id': 'b4e9e907-686e-440d-bc1a-d73a8decfa54', 'flavor_id': 'flavor-1-1-1', 'server_name': 'wuzs_mVCzIRxaJcOhK85ANTy9', 'server_id': '952e41e1-bf51-4648-af18-d59ca5d83557', 'project_token_name': 'KPYIfUOQAJdSBT7s41cM', 'admin_id': '0ec82b7380cb4d0c80058501938cdf48', 'admin_role_id': 'cc33f2c3a47247329d0ee7043c4f475f', 'volume_name': 'volume_oDk3NrBh1yWmOfeZuaJg', 'volume_id': '966dbf91-6235-4dcc-a30f-05a956b7a173', 'server_snap_name': 'snap_8JkrX4yjSduLBPHcmTxC', 'server_snap_id': '', 'volume_snapshot_id': '2cd6c8e7-0a47-4d18-b4b7-e445790b6b6f', 'network_name_for_check': '5yP1bXxrJVpowg8G2EsT_netwrok'}
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]====================================================================================================
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:23:04]请求头处理结果：  {'Content-Type': 'application/json', 'X-Auth-Token': '14be2548e67d4e1986d1ad542dfba7cf'}
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:23:04]请求参数处理结果：  {'Content-Type': 'application/json', 'X-Auth-Token': '14be2548e67d4e1986d1ad542dfba7cf'}
DEBUG    root:Log.py:73 [DEBUG 2019-11-03 20:23:04]host处理结果：  192.168.54.120
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]请求接口：删除网络后查看网络
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]请求地址：http://192.168.54.120:9696/v2.0/networks/af23f0b5-978f-4f65-9c8c-ed7685b92bc4
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]请求头: {'Content-Type': 'application/json', 'X-Auth-Token': '14be2548e67d4e1986d1ad542dfba7cf'}
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]请求参数: {'subnet': {'admin_state_up': True}}
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]GET请求接口
DEBUG    urllib3.connectionpool:connectionpool.py:205 Starting new HTTP connection (1): 192.168.54.120:9696
DEBUG    urllib3.connectionpool:connectionpool.py:393 http://192.168.54.120:9696 "GET /v2.0/networks/af23f0b5-978f-4f65-9c8c-ed7685b92bc4?subnet=admin_state_up HTTP/1.1" 404 137
INFO     root:Log.py:79 [INFO 2019-11-03 20:23:04]接口请求结果：
                         (404, {'NeutronError': {'message': 'Network af23f0b5-978f-4f65-9c8c-ed7685b92bc4 could not be found', 'type': 'NetworkNotFound', 'detail': ''}})